examples/android/app/src/main/jni/main.cpp

```diff
@@ -86,9 +86,7 @@ int main(int argc, char* argv[])
     sf::RenderWindow window(screen, "");
     window.setFramerateLimit(30);
 
-    sf::Texture texture;
-    if (!texture.loadFromFile("image.png"))
-        return EXIT_FAILURE;
+    const auto texture = sf::Texture::loadFromFile("image.png").value();
```

Comment:
This changes the behavior to throwing an exception rather than returning `EXIT_FAILURE`. Given that some behavior changes ended up being reverted in #3051, do we want to maintain the same behavior as before here too?

Reply:
> This changes the behavior to throwing an exception

In practice the result is the same. Failure to load means the program stops immediately. 

> Given that some behavior changes ended up being reverted in https://github.com/SFML/SFML/pull/3051, do we want to maintain the same behavior as before here too?

#3051 was different. In that case I truly did change behavior in a way that totally broke the example on certain platforms. For now I want to use the same `.value()` pattern in our examples until we come to a consensus on a better alternative. I'm not sure exactly what alternative you or others may prefer.

Reply:
Makes sense, you are correct that #3051 was a special case. Still, would be nice to have a policy regarding exceptions in our examples. Resolving for now.